schema.yaml

```diff
@@ -142,6 +142,23 @@ properties:
           patternProperties:
             "^power|fastboot_key|power_key|usb_disconnect$":
               $ref: "#/$defs/local_gpio"
+
+        laurent:
+          description: KernelChip Laurent relays
+          type: object
+          unevaluatedItems: false
+          properties:
+            server:
+              type: string
+            relay:
```

Comment:
Is this the relay port number ? Perhaps use something like "port" instead

Reply:
I don't want for it to be confused with the TCP port on which to talk to the server.